src/ignitus-UserInterfaceBook/Components/Organisms/interfaceNotifications/style.ts

```diff
@@ -0,0 +1,35 @@
+import styled from '@emotion/styled';
+import { Notifications } from '../../../../ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Notifications/Components';
+import { Icon } from '../../../../ignitus-Shared/ignitus-Utilities/Components/icon';
+import { Interface } from '../../../styles';
+
+export const NavigationContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex: 1;
+`;
+
+export const StyledNotifications = styled(Notifications)`
+  height: 2rem;
+  width: 70%;
+  padding: 1rem;
+`;
+
+export const Text = styled.span`
```

Comment:
For the normal text use the paragraph tag from the styling and for bold choose from the defined typography.